cache/sampleStore.js

```diff
@@ -30,7 +31,7 @@ const constants = {
       'okRange',
       'writers',
     ],
-    sample: ['relatedLinks'],
+    sample: ['relatedLinks', 'user'],
```

Comment:
Looks like users are stored as providers for samples. Should this be provider or user?